connector/github/github.go

```diff
@@ -68,10 +108,20 @@ func (c *githubConnector) oauth2Config(scopes connector.Scopes) *oauth2.Config {
 	} else {
 		githubScopes = []string{scopeEmail}
 	}
+
+	var endpoint oauth2.Endpoint
```

Comment:
nit: you can avoid the "else" condition by initializing this with the "else" value.